ToDoList/src/controller/Sort.java

```diff
@@ -0,0 +1,25 @@
+package controller;
+
+/**
+ * Tells ToDoList how to sort itself
+ * 
+ * @author Krishna Sannasi
+ *
+ */
+public enum Sort

```

Comment:
I am unsure what code will go in this part. Will we write code to sort in these different ways in this file, or is this simply used to distinguish a difference between the sorting options.

Reply:
This is only used to distinguish between the different options.